cli/dcoscli/config/main.py

```diff
@@ -86,28 +85,18 @@ def _set(name, value):
     """
 
     if name == "core.dcos_url":
-        return _cluster_setup(value)
+        notice = (
+            "This config property is being deprecated. "
+            "To setup the CLI to talk to your cluster, please run "
+            "`dcos cluster setup <dcos_url>`.")
+        emitter.publish(DefaultError(notice))
```

Comment:
`config` will point to the current active cluster, if there is none it will just error out with `No cluster is attached. Please run dcos cluster attach <cluster-name>`. So it cannot work if there is no cluster setup.

I think the DCOS_CONFIG stuff is also deprecated? (https://github.com/dcos/dcos-cli/pull/943). It seems like setting-up a cluster is the only way to go now, if I cannot run the cluster setup for CI I'd have to find a way to create manually the structure folder (with the "attached" file etc.).